tools/ReleaseTools.psm1

```diff
@@ -108,7 +108,9 @@ function Get-Bullets {
             'resolved'
         )
 
-        $IssueRegex = '(' + ($CloseKeywords -join '|') + ')\s+(?<repo>\D+)(?<number>\d+)'
+        # NOTE: The URL matcher must be explicit because the body of a PR may
+        # contain other URLs with digits (like an image asset).
+        $IssueRegex = '(' + ($CloseKeywords -join '|') + ')\s+((https://github.com/PowerShell/(?<repo>(' + ([RepoNames]::Values -join '|') + '))/issues/)|#)(?<number>\d+)'
```

Comment:
what case does the `|#` account for after the /issues/?

Reply:
The case where a URL was not used an instead just `#1234`. So it's either `the/url/issues/1234` OR `#1234`.